backend/app/services/resource_library/service.py

```
@@ -0,0 +1,741 @@
+        db: Session,
+        *,
+        listing_id: int,
+        user_id: int,
+        include_archived_for_owner: bool = False,
+    ) -> Kind:
+        listing = (
+            db.query(Kind)
+            .filter(
+                Kind.id == listing_id,
+                Kind.kind == RESOURCE_LIBRARY_KIND,
+                Kind.is_active == True,
```

Ruff (0.15.15)

[error] Avoid equality comparisons to `True`; use `Kind.is_active:` for truth checks

Replace with `Kind.is_active`

(E712)